app/src/main/java/a3/com/convo/activities/PlayGameActivity.java

```diff
@@ -17,17 +17,40 @@
 
     private FragmentTransaction ft;
 
+    // need as an instance in order to save state on configuration change to landscape mode
+    private GameFragment gameFrag;
```

Comment:
Let us NOT save fragment in memory. You can get saved state from the activity lifecycle